tests/rl/test_predictor_rl.py

```diff
@@ -14,7 +14,7 @@
 def test_compile(opt_objective):
     predictor = rl.Predictor()
     qc = get_benchmark("ghz", 1, 5)
-    qc_compiled, device = predictor.compile(qc, opt_objective=opt_objective)
+    qc_compiled, device = predictor.compile_prediction(qc, opt_objective=opt_objective)
```

Comment:
Hm. thinking about it: For the RL predictor `compile_prediction` does not really make sense, since there is no real prediction involved. (which is also reflected in the corresponding function)

Would it make sense to just call both methods (RL and ML) `compile`?
In the very end, both take a circuit, query the ml/rl-model, and return the compiled circuit.

Reply:
The reason I wanted to avoid `compile` as a function name is, that there is a python built-in function with the same name. To avoid confusion (and, at least in my setup, strange confusing code highlighting), I changed the name.

Reply:
Ah, I see. Now I remember..
hm. Maybe that's why Qiskit is using `transpile` ;o) Although I don't like that term.
I feel `compile_qc` is already heading in the right direction although a single word name would be even better. 
But I can't really think of anything clever.

Reply:
Both methods are now renamed to `qcompile`.